pkg/continuoustest/client.go

```diff
@@ -67,7 +70,7 @@ func (cfg *ClientConfig) RegisterFlags(f *flag.FlagSet) {
 
 	f.Var(&cfg.ReadBaseEndpoint, "tests.read-endpoint", "The base endpoint on the read path. The URL should have no trailing slash. The specific API path is appended by the tool to the URL, for example /api/v1/query_range for range query API, so the configured URL must not include it.")
 	f.DurationVar(&cfg.ReadTimeout, "tests.read-timeout", 60*time.Second, "The timeout for a single read request.")
-
+	f.BoolVar(&cfg.RequestDebug, "tests.request-debug", false, "Request debugging on the server side via header.")
```

Comment:
[nit] How about something like `-tests.request-chunks-debugging`? Then it's very clear exactly what this enables.

As it stands, from the name alone, it sounds like continuous-test will log more information about the requests it sends.

Reply:
maybe be more explicit and have `-tests.add-debug-header` ?

Reply:
How about `-tests.send-chunks-debugging-header`?